int32 object_id
geometry_msgs/Pose object_pose
float32[] pregrasp_offsets
int32 num_configurations
---
srs_msgs/DBGrasp[] side
srs_msgs/DBGrasp[] mside
srs_msgs/DBGrasp[] front
srs_msgs/DBGrasp[] top
